src/components/structures/ScrollPanel.js

```diff
@@ -332,28 +385,34 @@ module.exports = React.createClass({
 
             var boundingRect = node.getBoundingClientRect();
             if (boundingRect.bottom < wrapperRect.bottom) {
-                return {
-                    atBottom: atBottom,
+                this.scrollState = {
+                    atBottom: false,
```

Comment:
Are we safe doing this? Previously, this would occasionally be `true` if `this.isAtBottom()` was `true`, which makes sense. Now we're just bluntly setting it to `false` for.. reasons? The guard on `:372` doesn't work if `this.props.stickyBottom` is `false`. Do we really mean `this.scrollState.atBottom` is `atStickyBottom`? I don't follow why this is always `false`.